Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -78,6 +78,13 @@ def bump_formula_pr_args
                           "or specified <version>."
       switch "-f", "--force",
              description: "Ignore duplicate open PRs. Remove all mirrors if `--mirror` was not specified."
+      flag   "--python-package-name=",
+             description: "Use the specified <package-name> when finding Python resources for <formula>. "\
+                          "If no package name is specified, it will be inferred from the formula's stable URL."
+      comma_array "--python-extra-packages=",
+                  description: "Include these additional Python packages when finding resources."
+      comma_array "--python-exclude-packages=",
+                  description: "Exclude these Python packages when finding resources."
```

Comment:
Not sure what the preferred flag ordering is here (if the new args need to be inserted somewhere else in the args list)

Also went with `--python-<flag>` naming instead of the original suggested `--extra-python-packages` for consistency (python in the middle works for include/exclude but not `--package-name`)

Reply:
Just to check: do you personally need all three of these arguments?

Reply:
Yes, because the way that the existing code in `PyPI.update_python_resources` only loads the contents of `pypi_formula_mappings.json` if none of these flags are specified via the CLI.

https://github.com/Homebrew/brew/blob/ce5cb9b745e4107d159f57ec031f2bac033e5686/Library/Homebrew/utils/pypi.rb#L141-L143

So in order to get the behavior I'm looking for (to build DVC with additional `--python-extra-packages` hints to be passed into pipgrip), I have to also specify the package name + excludes via the CLI (even though they are listed in `pypi_formula_mappings.json`)